data/processed/defra/mean_pm10.yaml

```diff
@@ -0,0 +1,18 @@
+API: DEFRA
+API documentation: https://uk-air.defra.gov.uk/data/pcm-data
+Dataset code: mean_pm10_air_pollution
+endpoint: https://uk-air.defra.gov.uk/data/pcm-data#pm10
+description: Mean PM10 particulate background pollution data at NUTS 2 regions aggregated from 1km x 1km resolution data modelled by DEFRA.
+nuts_version: 2016
+schema:
+  nuts_id:
+    type: string
+    description: NUTS2 (2016) region code

```

Comment:
since we're using `nuts_version` maybe we can remove the year in the description, one less thing to maintain should be update the NUTS version